locale/en/blog/weekly-updates/weekly-update.2016-04-29.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Weekly Update - Apr 29th, 2016
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-04-29T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-04-29
+layout: blog-post.hbs
+---
+
+### Node v6.0.0 (Current) Release
+
+[Node v6.0.0 (Current)](https://nodejs.org/en/blog/release/v6.0.0/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```

Comment:
Not sure if this would be helpful as well: https://github.com/nodejs/node/wiki/Breaking-changes-between-v5-and-v6